<HTML><HEAD><TITLE>IMEx Results HomePage</TITLE><style type='text/css'>a{color: #A00000;font-size: 13px;font-family: Arial,lucida, Verdana, Geneva, Arial, Helvetica, sans-serif;text-decoration: none;}th {color : #ffffff;background-color:#A00000;font-size : 11px;font-family : Tahoma,Verdana,Arial;}td {color : #000000;font-size : 14px; font-family : Tahoma,Verdana,Arial;}</style></HEAD><BODY bgcolor='#F0F0E1'><center><table border='0' width='700' height='80' cellspacing='0' cellpadding='0' bgcolor='#A00000'><tr><td><font face='Arial' size='5' color='#FFFFFF'><center>IMEx: Imperfect Microsatellite Extractor 2.0</center></font><b><font face='Arial' color='#FFFFFF' size='1'><center>Program for extracting microsatellites from genome sequences</center></font></b></td></tr></table></center><br><font face='Arial' size='2'><font color='#A00000'>Sequence : </font>>NZ_CP022088.2 Nocardia brasiliensis strain FDAARGOS_352 chromosome, complete genome
 (8935513 bp)<br><style type='text/css'> #par {border-color:black;border-style:groove;}</style><table border='1' id='par' cellpadding='2' cellspacing='2'><tr><td id='par'>Parameter</td><td id='par'><font color='#A00000'>Mono</font></td><td id='par'><font color='#A00000'>Di</font></td><td id='par'><font color='#A00000'>Tri</font></td><td id='par'><font color='#A00000'>Tetra</font></td><td id='par'><font color='#A00000'>Penta</font></td><td id='par'><font color='#A00000'>Hexa</font></td></tr><tr align='center'><td id='par'><font color='#A00000'>Imperfection % (p %) : </font></td><td id='par'>10 %</td><td id='par'> 10 %</td><td id='par'> 10 %</td><td id='par'> 10 %</td><td id='par'> 10 %</td><td id='par'> 10 %</td></tr><tr align='center'><td id='par'><font color='#A00000'>Mismatch in Pattern : </font></td><td id='par'> 1</td><td id='par'> 1</td><td id='par'> 1</td><td id='par'> 2</td><td id='par'> 2</td><td id='par'> 2</td></tr><tr align='center'><td id='par'><font color='#A00000'>Repeat Number : </font></td><td id='par'> 12</td><td id='par'> 6</td><td id='par'> 4</td><td id='par'> 3</td><td id='par'> 3</td><td id='par'> 3</td></tr></table><font color='#A00000'>dMAX for cSSR:</font> 100 bp<br><font color='#A00000'>Standardization Level Selected:</font>Level 3 Standardization<br><b><br>PERFECT / IMPERFECT SSR RESULT:</b><br> (Click on the links inside the table below for results)<br><table border='1' bordercolor='black'><tbody><tr><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_1_summary.html'>1-1500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_2_summary.html'>1501-2500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_3_summary.html'>2501-4000 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_4_summary.html'>4001-4500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_5_summary.html'>4501-7500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_6_summary.html'>7501-8000 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_7_summary.html'>8001-9500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_8_summary.html'>9501-10500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_9_summary.html'>10501-11000 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_10_summary.html'>11001-12000 </a></td></tr><tr><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_1_summary.html'>12001-12500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_2_summary.html'>12501-13000 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_3_summary.html'>13001-13500 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_4_summary.html'>13501-14000 </a></td><td align='center'><a href='Nocardia_brasiliensis_FDAARGOS_352_15_summary.html'>14001-14477 </a></td></tr></tbody></table>Total No. of Microsatellites Extracted: 14477<br><br><font face='Arial' color='Black'><b>COMPOUND SSR OUTPUT:</b><br> (Click on the links inside the table below for results)</font><br><table border='1' bordercolor='black'><tbody><tr><td align='center'><a href='compound_summary_1.html'>1-500 </a></td><td align='center'><a href='compound_summary_2.html'>500-1000 </a></td><td align='center'><a href='compound_summary_3.html'>1000-1500 </a></td><td align='center'><a href='compound_summary_4.html'>1500-2000 </a></td><td align='center'><a href='compound_summary_5.html'>2000-2046 </a></td></tr></tbody></table>Total No. of Compound Microsatellites Extracted: 2046<br><br><font face='Arial' color='Black'><b>STANDARDIZATION RESULTS:</b><br></font><a href='ssr_statistics.html'>Click here for SSR Standardization</a><br><a href='compound_statistics.html'>Click here for Compound SSR Standardization</a><br><br><br><b>Please Cite:</b><br>Suresh B.Mudunuri and H.A.Nagarajaram.(2007)IMEx:Imperfect Microsatellite Extractor. <br><i>Bioinformatics</i> 23(10):1181-1187.<br><br><br><br><br></body></html>